<?xml version="1.0"?>

<launch>

  <node pkg="francor_geotiff" type="geotiff_node" name="francor_geotiff_node" output="screen" launch-prefix="nice -n 15">
    <!-- <remap from="map" to="scanmatcher_map" /> -->
    <param name="map_file_path" type="string" value="$(find francor_geotiff)/maps" />
    <param name="map_file_base_name" type="string" value="test_hans" />
    <param name="geotiff_save_period" type="double" value="0" />

    <param name="VictimMapWriter/draw_all_objects" value="false" />
    <param name="VictimMapWriter/class_id" value="none" />
    <param name="QRCodeMapWriter/draw_all_objects" value="false" />
    <param name="QRCodeMapWriter/class_id" value="none" />
  </node>

</launch>

